packages/vue-components/src/Navbar.vue

```diff
@@ -283,17 +288,22 @@ export default {
 
         div.navbar-brand {
             padding-left: 1rem;
-            max-width: 50%;
+            max-width: calc(50% - 1rem);
```

Comment:
should we define our own `.navbar-brand` (`.navbar-left`?) class of sorts for use here to avoid relying on `1rem` specifically?

I did a very brief look over the original definition and it seems like it only sets a few paddings / margins / font sizes, nothing too fancy we can't do ourselves 😃